content/api/hub/template_zola_ssg.md

```diff
@@ -0,0 +1,39 @@
+title = "Zola SSG Tempalte"
+template = "render_hub_content_body"
+date = "2023-09-08T00:00:00Z"
+content-type = "text/html"
+tags = ["rust", "SSG", "http"]
+
+[extra]
+author = "karthik2804"
+type = "hub_document"
+category = "Template"
+language = "rust"
```

Comment:
Looking at the content this only uses the static file server so I'm not sure how meaningful the language is.  We can tweak that later if need be though.